draft-ietf-ice-rfc5245bis.xml

```diff
@@ -1611,54 +1420,50 @@ of this specification.
 <section anchor="sec-forming" title="Forming the Check Lists">
 
 <t>
-There is one CHECK LIST per in-use media stream resulting from the
-candidate exchange.  To form the CHECK LIST for a media stream, the
-agent forms candidate pairs, computes a candidate pair priority,
-orders the pairs by priority, prunes them, removes lower-priority
-candidates and sets their states. These steps are described in this section.
-If a CHECK LIST is updated (e.g, due to detection of peer reflexive
-candidates), the agent will re-perform the steps for the updated
-CHECK LIST.
+There is one check list for each media stream.  To form a check list,
+an agent (initiaing and responding) forms candidate pairs, computes
+pair priorities, orders pairs by priority, prunes pairs, removes
+lower-priority pairs, and sets check list states. If candidates are
+added to a check list (e.g, due to detection of peer reflexive
+candidates), the agent will re-perform these steps for the updated
+check list.
 </t>
 <section title="Check List State">
-<t>Each CHECK LIST has a state, which captures the state of ICE checks
-for the media stream associated with the CHECK LIST. The states are:</t>
+<t>Each check list has one of the following states:</t>
 <t><list style="hanging">
-<t hangText="Running:"> In this state, ICE checks are still in
-progress for this media stream. Check lists are initially set
+
+<t hangText="Running:">The check list is neither Completed yet nor
+Failed yet.  Check lists are initially set
 to the Running state.
 </t>
-<t hangText="Completed:"> In this state, ICE checks have produced
-selected pairs for each component of the media stream.
+
+<t hangText="Completed:">The check list has a selected candidate pair
+for each component of the media stream.
 </t>
-<t hangText="Failed:"> In this state, the ICE checks have not
-completed successfully for this media stream.
+
+<t hangText="Failed:">The check list does not have a valid candidate
+pair for each component of the media stream and all of the candidate
+pairs in the check list are in either the Failed or Succeeded state.
+In other words, at least one component of the check list has candidate
+pairs that are all in the Failed state, which means the component has 
+failed, which means the check list has failed.
 </t>
+
 </list></t>
-<t>A CHECK LIST with at least one pair that is
-Waiting is called an active CHECK LIST, and a CHECK LIST with all
-pairs Frozen is called a frozen CHECK LIST.
+<t>Additionally, a check list with at least one pair in the waiting
+state is called "active", while a check list with all pairs in the
+frozen state is called "frozen".
 </t>
+
 </section>
 
 <section title="Forming Candidate Pairs">
 
 <t>
-First, the agent takes each of its candidates for a media stream
```

Comment:
Sorry, I thought I re-added text here.  I guess not.  I just did so again.  It's almost the same as the original, so I'm not sure it's worth discussing.